src/test/java/teammates/test/pageobjects/IndexPage.java

```diff
@@ -0,0 +1,15 @@
+package teammates.test.pageobjects;
+
+public class IndexPage extends AppPage {
+
+    public IndexPage(Browser browser) {
+        super(browser);
+    }
+
+    @Override
+    protected boolean containsExpectedPageContents() {
+        String pageSource = getPageSource();
+        return pageSource.contains("Student Login") && pageSource.contains("Instructor Login");
```

Comment:
I think this might be a very precarious test condition. Isn't there any other string indicative of the index page? Worst case scenario, we can plant some unique string that can be used to uniquely identify the page.